charts/timescaledb-single/values.yaml

```diff
@@ -19,6 +19,9 @@ image:
   tag: pg11-ts1.6
   pullPolicy: IfNotPresent
 
+securityContext:
+  runAsUser: 1000
```

Comment:
```suggestion
# Some documentation about to what purpose this serves
# https://example.com/...
securityContext:
  enabled: false
  runAsUser: 1000
```

would be good to have some documentation as to when and why this would be good

Reply:
Will do later today.